web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -231,11 +231,20 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
 
   activeGestures: GestureHandler[] = [];
   activeModipress: Modipress = null;
+  private _deferLayout: boolean;
 
   // The keyboard object corresponding to this VisualKeyboard.
   public readonly layoutKeyboard: Keyboard;
   public readonly layoutKeyboardProperties: KeyboardProperties;
 
+  get deferLayout(): boolean {
+    return this._deferLayout;
+  }
+
+  set deferLayout(value: boolean) {
+    this._deferLayout = value;
+  }
```

Comment:
Why does _deferLayout need to be private given the getter and setter do nothing?

Reply:
Artifact from development I should clean up.

At one point, I thought the setter would be needed, in which case turning it off via assignment would have called `.layoutIfNeeded()`.